examples/main.go

```diff
@@ -237,15 +233,15 @@ func (c *Counter) HandleEvent(ctx context.Context, e *live.Event) error {
 }
 
 func (c *Counter) Render(ctx context.Context, meta *live.Meta) (any, *htmltmpl.Template) {
-	return c, htmltmpl.Must(htmltmpl.New("liveView").Funcs(myFuncs).Parse(`
-			{{ define "inputTag" }}
-				<input type="text" name="{{ .Name }}" value="{{ .Changeset.Value .Name }}"/>
-			{{ end }}
-			{{ define "errorTag" }}
-				{{ if .Changeset.HasError .Name}}
-					<span class="error">{{ .Changeset.Error .Name }}</span>
-				{{ end }}
-			{{ end }}
+	funcs := htmltmpl.FuncMap{
```

Comment:
Oh, I assume because we don't know the generic type until runtime

Reply:
this seemed like the only way to add typed `Changeset` specific funcs without resorting to some naming convention that explodes with the number of typed `Changesets`.  LMK if you have any better ideas.